AtDigitization/AtVectorResponse.cxx

```diff
@@ -0,0 +1,49 @@
+#include "AtVectorResponse.h"
+
+#include "AtDigiPar.h"
+
+#include <FairRunAna.h>
+#include <FairRuntimeDb.h>
+#include <FairTask.h>
+
```

Comment:
Should add the following:
![image](https://user-images.githubusercontent.com/13934511/180503650-8149928e-ce0c-493f-a73a-e2a8f50d019c.png)